src/service-worker.ts

```diff
@@ -98,97 +86,61 @@ export async function generateServiceWorkerConfig(
   options = Object.assign({}, options);
   const project = options.project;
   const buildRoot = options.buildRoot;
-  const swPrecacheConfig: SWConfig =
-      Object.assign({}, options.swPrecacheConfig);
+  const workboxConfig: WorkboxConfig =
+    Object.assign({}, {
+      globDirectory: buildRoot,
+      importScripts: [getModuleUrl('workbox-sw')],
```

Comment:
Thanks for moving this into the `Object.assign()` logic.

In the back of my head, I'm wondering whether something special needs to be done for `importScripts`, apart from just overwriting the default `importScripts` array (which contains the Workbox CDN URL).

For instance, if `options.workboxConfig.importScripts === ['my-push-hander.js']`, I think the developer's intention is that they still want Workbox from the CDN, but they also want to load in their custom push handler script. With the code as it is right now, they'd just end up with the push handler and no Workbox CDN.

If we assume that the developer will *always* load Workbox from the CDN URL, then instead of overwriting the default `importScripts` in this `Object.assign()`, you could instead do

```js
workbox.config.importScripts =[getModuleUrl('workbox-sw')];
if ('importScripts' in options. workboxConfig) {
  workbox.config.importScripts = workbox.config.importScripts.concat(
    options.workboxConfig.importScripts);
}
```

Or something like that. It might not perfectly capture all use cases, but I think it gives some more flexibility—and if we get folks who ask for non-CDN builds, that would involve re-evaluating this anyway.